Dockerfile

```diff
@@ -1,11 +1,6 @@
-FROM bash AS build
-
-COPY install-php-extensions /tmp/install-php-extensions
-RUN chmod +x /tmp/install-php-extensions
-
 FROM scratch
 
-LABEL org.opencontainers.image.source="https://github.com/mlocati/docker-php-extension-installer" \
-      org.opencontainers.image.licenses="MIT"
+LABEL org.opencontainers.image.source="https://github.com/mlocati/docker-php-extension-installer"
```

Comment:
@mlocati 

The official Docker documentation says that it is no longer necessary to write multiple labels on a single line.

https://docs.docker.com/build/building/best-practices/#label
> An image can have more than one label. Prior to Docker 1.10, it was recommended to combine all labels into a single LABEL instruction, to prevent extra layers from being created. This is no longer necessary, but combining labels is still supported.

I think splitting them up makes the Dockerfile structure clearer and easier to maintain. This also applies to the `ENV` instruction.

Reply:
Thanks!